x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/components/search_ui_form.tsx

```diff
@@ -126,6 +131,19 @@ export const SearchUIForm: React.FC = () => {
           data-test-subj="selectUrl"
         />
       </EuiFormRow>
+      <EuiFormRow label={THUMBNAIL_FIELD_LABEL} helpText={THUMBNAIL_FIELD_HELP_TEXT} fullWidth>
+        <EuiSelect
+          disabled={dataLoading}
+          options={optionFields}
+          value={selectedThumbnailOption && selectedThumbnailOption.value}
```

Comment:
Just to check, we don't anticipate adding a default value for this the same way we do for `title` and `url` right? Might be worth checking with the Crawler folks perhaps to see if they scrape a page thumbnail image?...

Reply:
I can check with them

Reply:
Will add defaulting to a future PR.